src/agent/agent.service.ts

```diff
@@ -78,6 +78,16 @@ export class AgentService {
         return await this.agentCaller.callAgent(process.env.AGENT_ID, process.env.ADMIN_API_KEY, 'POST', 'wallet/did/public', params);
     }
 
+    /**
+     * Deletes credential using the cred_rev_id for issuer
+     */
+     public async deleteCredential(credential_id : string): Promise<any> {
+        const data = {
```

Comment:
you don't need to pass data, instead you just pass in credId directly in the url, similar to https://github.com/kiva/aries-controller/blob/main/src/issuer/issuer.service.ts#L249